packages/sil-governance/src/routes/create-proposal/components/ProposalForm.tsx

```diff
@@ -142,6 +143,16 @@ const ProposalForm = (): JSX.Element => {
     const createProposalTx = await governor.buildCreateProposalTx(proposalOptions);
 
     const transactionId = await sendSignAndPostRequest(connection, createProposalTx);
+
+    const updateChainProposals = async (): Promise<void> => {
+      const chainProposals = await getProposals(governor);
+      dispatch(replaceProposalsAction(chainProposals));
+    };
+
+    setTimeout(() => {
+      updateChainProposals();
+    }, 5000);
```

Comment:
Suggestion: this can be written as a on-liner:

```ts
setTimeout(() => updateChainProposals(), 5000);
```

Reply:
Sure! There were more lines before